src/components/Table.tsx

```diff
@@ -104,9 +106,8 @@ const TableColumns = styled.div<{ columnWidths?: (string | undefined)[] }>`
           grid-auto-flow: column;
         `};
 
-  padding: 0 var(--spacing-5);
   align-items: center;
-  height: var(--tableCellHeight);
+  padding: 17.5px 20px;
```

Comment:
`.5px`? 😅

Reply:
It was the distance in the mock :shrug:

Reply:
Haha, as I said, the mockups are kind of alive and not 100% pixel perfect. I would spend much more time and be much less creative if I wanted them to be absolutely perfect all the time.

Let's round values when this happens.

Reply:
ping @LeeAlephium

Reply:
Yep, all good! It was an answer to the "why" - There was a PR when I first started where the mocks were to be followed very strictly, and there were a few times when it wasn't, so I've learned from then to be more strict first and then re-adjust as needed in the PR review